src/animators/baseAnimator.ts

```diff
@@ -63,7 +63,7 @@ export module Animators {
       return time;
     }
 
-    public animate(selection: any, attrToProjector: AttributeToProjector): D3.Transition.Transition {
+    public animate(selection: d3.Selection<any>, attrToAppliedProjector: AttributeToAppliedProjector) {
```

Comment:
Can you tell me a bit about the difference between `AttributeToProjector` and `AttributeToAppliedProjector`